bin/generate-wallet-connect-key/index.ts

```diff
@@ -0,0 +1,24 @@
+import dotenv from 'dotenv'
+import { mkdir, writeFile } from 'fs/promises'
+import { join, resolve } from 'path'
+
+dotenv.config({ path: resolve(__dirname, `../../.env.production`) })
+
+export async function main() {
+  const directoryAbsolutePath = './public/.well-known'
```

Comment:
why the directory will be called `.well-known` ?

Reply:
That's a requirement from the verifier! they set what path should the file be in